src/main/java/com/autotune/utils/KruizeConstants.java

```diff
@@ -189,6 +189,7 @@ public static final class JSONKeys {
         public static final String INTERVAL_START_TIME = "interval_start_time";
```

Comment:
Is this file really needed for this PR? Not really important but this seems to be part of the perf profile changes